packages/angular_devkit/build_angular/src/karma/index.ts

```diff
@@ -95,6 +111,33 @@ export function execute(
         }
       }
 
+      // prepend special webpack loader that will transform test.ts
+      if (webpackConfig && webpackConfig.module && options.spec) {
+        const sourceRoot = workspace.getProject(projectName).sourceRoot || '';
+        const files = findSpecs(options.spec, workspace.root, sourceRoot);
```

Comment:
I'll see if I can add array support - could be beneficial for CI environment to run only diff